apps-rendering/src/components/standfirst.stories.tsx

```diff
@@ -1,10 +1,15 @@
 // ----- Imports ----- //
 
-import { ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
 import { boolean, withKnobs } from '@storybook/addon-knobs';
 import { article, comment, feature, review } from 'fixtures/item';
+import { deadBlog } from 'fixtures/live';
 import type { ReactElement } from 'react';
 import { selectPillar } from 'storybookHelpers';
+import {
+	getAllThemes,
+	getThemeNameAsString,
+} from '../../../common-rendering/src/fixtures/article';
```

Comment:
I think you can import this with the `guardian` prefix if you want? See `headerImage.tsx` for an example.